apps/src/netsim/NetSimSendWidget.js

```diff
@@ -54,24 +86,123 @@ NetSimSendWidget.createWithin = function (element, connection) {
 NetSimSendWidget.prototype.bindElements_ = function () {
   this.rootDiv_ = $('#netsim_send_widget');
   this.toAddressTextbox_ = this.rootDiv_.find('#to_address');
-  this.payloadTextbox_ = this.rootDiv_.find('#payload');
+  this.toAddressTextbox_.bind('keyPress', function (event) {
```

Comment:
Though I don't think it's necessarily a reserved word, I think IE treats "event" specially, and this might give you problems.